src/components/templates/AnimateMap/hooks/useRelatedPartymapRooms.ts

```diff
@@ -33,7 +34,7 @@ export const useRelatedPartymapRooms: (
       relatedPartymap?.rooms?.map((portal) => {
         const noTrailSlashPortalUrl = getUrlWithoutTrailingSlash(portal.url);
 
-        const [venueId] = getLastUrlParam(noTrailSlashPortalUrl);
+        const [venueId] = getLastUrlParam(noTrailSlashPortalUrl) as SpaceId[];
         const portalVenue = findVenueInRelatedVenues({ spaceId: venueId });
```

Comment:
[OOS]
I had a refactor of this function elsewhere in larger PR that got rolled back. It would have been nice to have it back